content/en/docs/instrumentation/net/resources.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "Resources"
+weight: 6
+description:
+---
+
+A [resource][] represents the entity producing telemetry as resource attributes.
+For example, a process producing telemetry that is running in a container on
+Kubernetes has a Pod name, a namespace, and possibly a deployment name. All
+three of these attributes can be included in the resource.
+
+In your observability backend, you can use resource information to better
+investigate interesting behavior. For example, if your trace or metrics data
+indicate latency in your system, you can narrow it down to a specific container,
+pod, or kubernetes deployment.
+
+## Setup
+
+Follow the instructions in the [Getting Started][], so that you have a running
+.NET app exporting data to the console.
+
+## Adding resources with environment variables
+
+You can use the `OTEL_RESOURCE_ATTRIBUTES` environment variable to inject
```

Comment:
I filed https://github.com/open-telemetry/opentelemetry.io/issues/1830 for this -- for now I'll keep this since it's also in the JS doc, and we can address these uniformly.